backbone/src/test/java/org/researchstack/backbone/task/factory/TappingTaskTest.java

```diff
@@ -31,8 +27,8 @@
 
 public class TappingTaskTest {
 
-    private Context mockContext;
-    private Resources mockResources;
+    @Mock private Context mockContext;
```

Comment:
To get these annotations to auto-mock, you need to either annotate the Test class with @RunWith(MockitoJUnitRunner.class) or call MockitoAnnotations.initMocks(this); before every test